api/organisations/models.py

```diff
@@ -157,6 +157,20 @@ def rebuild_environments(self):
         ).values_list("id", flat=True):
             rebuild_environment_document.delay(args=(environment_id,))
 
+    def cancel_users(self):
+        remaining_seat_holder = (
+            UserOrganisation.objects.filter(
+                organisation=self,
+                role=OrganisationRole.ADMIN,
+            )
+            .order_by("date_joined")
+            .first()
+        )
```

Comment:
@dabeeeenster to confirm as per the related issue (#2197), the logic we are implementing is such that we leave the first organisation admin that joined the organisation as the only user left on cancellation. In nearly all cases I imagine this will be the user that created the organisation.

Reply:
Yes this is fine